src/components/Badge/Badge.tsx

```diff
@@ -28,17 +28,18 @@ const Badge = React.forwardRef<HTMLElement, BadgeProps>(function Badge(
       aria-atomic="true"
       ref={ref}
       cursor="default"
-      width="fit-content"
-      minWidth="85px"
+      width={size === 'medium' ? 'fit-content' : 'auto'}
       textAlign="center"
-      fontWeight="bold"
-      borderRadius="pill"
+      fontWeight={size === 'medium' ? 'normal' : 'bold'}
```

Comment:
I don't think that `size` warrants this change here. Could it be that `size` is the wrong prop to describe this variation?

Reply:
You are absolutely right!